### PR TITLE
Rename va-featured-content to va-summary-box

### PR DESCRIPTION
**Pull request by ataker**
## Summary
Design system is renaming the component va-featured-content to va-summary-box. This PR is to update current instances to the new name. No functional changes have been made.

## Related issue(s)

-[Update naming to Summary Box #2336](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2336)

## Testing done

- Tested updates locally in chrome

## Screenshots

Screenshot 2024-02-10 at 19 37 20
Screenshot 2024-02-10 at 19 34 28
Screenshot 2024-02-10 at 19 25 02
Screenshot 2024-02-10 at 19 24 48

## What areas of the site does it impact?

None outside of these files

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -2769,13 +2769,13 @@
   resolved "https://registry.yarnpkg.com/@datadog/sketches-js/-/sketches-js-2.1.0.tgz#8c7e8028a5fc22ad102fa542b0a446c956830455"
   integrity sha512-smLocSfrt3s53H/XSVP3/1kP42oqvrkjUPtyaFd1F79ux24oE31BKt+q0c6lsa6hOYrFzsIwyc5GXAI5JmfOew==
 
-"@department-of-veterans-affairs/component-library@^35.0.0":
-  version "35.0.0"
-  resolved "https://registry.yarnpkg.com/@department-of-veterans-affairs/component-library/-/component-library-35.0.0.tgz#6d204a8a90fa9a1a4df98d2d21481b2214e0d3be"
-  integrity sha512-oNJr2Ywzv5Aya5Z9ksAqHLLNI8XkTy8qWjZQDn/P0dUvWNq8oxBVPAaMxGLkR2oPc9iiZchmMd//up7NWKBWTA==
+"@department-of-veterans-affairs/component-library@^37.0.0":
+  version "37.0.0"
+  resolved "http://localhost:4873/@department-of-veterans-affairs%2fcomponent-library/-/component-library-37.0.0.tgz#b36cce43dc1d53b0f0fd55c496b5ddded5035e69"
```
Comment (micahchiang) [inline yarn.lock:2774]: This needs to be reset to the registry.yarnpkg.com

File: yarn.lock [hunk 2]
```diff
@@ -2821,10 +2821,10 @@
     yeoman-generator "^5.6.1"
     yosay "^2.0.2"
 
-"@department-of-veterans-affairs/react-components@26.0.1":
-  version "26.0.1"
-  resolved "https://registry.npmjs.org/@department-of-veterans-affairs/react-components/-/react-components-26.0.1.tgz#38eeb0dbe14ffe5f4a5fad6007d4688b33eabfdf"
-  integrity sha512-qCT32ZH7m6B5QMScML+dd/urB4IVNHmiJfX2XTFNE2ROzi2C61A7XMtDEt4bILa1Dno4H49QVhCFe3DqV6rU4w==
+"@department-of-veterans-affairs/react-components@28.0.0":
+  version "28.0.0"
+  resolved "http://localhost:4873/@department-of-veterans-affairs%2freact-components/-/react-components-28.0.0.tgz#8ff7bbc1303a68e7ad2806e743a8ee72416aa284"
```
Comment (micahchiang) [inline yarn.lock:2826]: This needs to be reset to the registry.yarnpkg.com

File: yarn.lock [hunk 3]
```diff
@@ -2862,10 +2862,10 @@
   dependencies:
     minimist "^1.2.6"
 
-"@department-of-veterans-affairs/web-components@4.59.1":
-  version "4.59.1"
-  resolved "https://registry.yarnpkg.com/@department-of-veterans-affairs/web-components/-/web-components-4.59.1.tgz#295cf8c2f04f1913ceef0ecffe0da39a3b38616c"
-  integrity sha512-nAc/ER8voz0uVzOdKylqmH0A9v7Kv7k8ukbho0SjfVnDH09QcCGxcMkD8k+WIVYTW2pQBUytp4rgmV/x+GiJ1Q==
+"@department-of-veterans-affairs/web-components@5.0.0":
+  version "5.0.0"
+  resolved "http://localhost:4873/@department-of-veterans-affairs%2fweb-components/-/web-components-5.0.0.tgz#81628780cef1e647dfd3f87d1473058b8cee481d"
```
Comment (micahchiang) [inline yarn.lock:2867]: This needs to be reset to the registry.yarnpkg.com

**Review by micahchiang — CHANGES_REQUESTED** [review]
Looks good, but I think your registry needs to be reset to yarn from verdaccio.

**Review by micahchiang — DISMISSED** [review]

**Review by adamwhitlock1 — DISMISSED** [review]
lgtm for authenticate experience! thanks 🚀

**Review by Andrew565 — DISMISSED** [review]

**Comment by micahchiang** [comment]
@ataker - just FYI, there's a PR out that includes a `va-featured-content` component. If it gets merged, please be on the look for it when you merge `main` into this branch. https://github.com/department-of-veterans-affairs/vets-website/pull/28007

**Review by Mottie — DISMISSED** [review]

**Review by Mottie — DISMISSED** [review]

**Review by mchae-nava — APPROVED** [review]
LGTM

**Review by micahchiang — APPROVED** [review]

**Review by adamwhitlock1 — APPROVED** [review]

**Review by Mottie — APPROVED** [review]

**Comment by micahchiang** [comment]
Someday, we'll get this thing merged :rage1:

**Pull request merged**